src/Talk/Resources/config/services.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  _instanceof:
+    App\Action:
+      tags: [controller.service_arguments]
```

Comment:
I just tried it as I was unsure, turns out since only $request (SF HttpFoundation request) is passed in, you do not need that tag.
Makes me wonder whether we need an action interface... The container and dispatcher surely do not care abut that in SF...

Reply:
I also thought I could remove the tag at first, but it didn't work. Symfony automatically discards every unused private service from the container, and since actions are never injected anywhere the get discarded. We could just set actions as public without needing the tag, but for that we would still need the interface